resolvers/event.js

```diff
@@ -62,27 +63,33 @@ const eventResolvers = {
     },
     updateEvent: async (parent, args) => {
       // Checking if event is valid
-      const event = await db.event.findByPk(args.id);
-      if (!event) {
-        throw new Error("Invalid event ID");
-      }
+      await db.event.findByPk(args.id).then((event) => {
+        if (!event) {
+          throw new Error('Invalid event ID: ' + args.id);
+        }
+      });
 
       // Checking if user is valid
       if (args.createdBy) {
-        const user = await db.user.findByPk(args.createdBy);
-        if (!user) {
-          throw new Error("Invalid user ID");
-        }
+        await db.user.findByPk(args.createdBy).then((user) => {
+          if (!user) {
+            throw new Error('Invalid user ID: ' + args.createdBy);
+          }
+        });
       }
 
-      const addAmbulances = async () => {
-        // Checking if ambulances are valid
-        for (const ambulanceId of args.ambulances) {
-          const ambulance = await db.ambulance.findByPk(ambulanceId["id"]);
-          if (!ambulance) {
-            return true;
-          }
-        }
+      if (args.ambulances) {
+        // Checking if all ambulances exist
+        await Promise.all(
```

Comment:
this syntax makes sure all promises (i.e. `db` calls) returned from the map resolve first (via await) before proceeding